Unix/codec/mof/parser/utility.c

```diff
@@ -1092,13 +1023,8 @@ void ReportFinalError(
     switch (id)
     {
     case ID_OUT_OF_MEMORY:
-#if defined(_MSC_VER)
-        errorCode = (MI_Uint32)E_OUTOFMEMORY;
-        errorType = MI_RESULT_TYPE_HRESULT;
-#else
         errorCode = ENOMEM;
         errorType = MI_T("STDC"); /* todo: Fixup error type for non-win */
```

Comment:
um, I believe this should be MI_RESULT_TYPE_ERRNO

Reply:
Changed.